cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -196,6 +196,12 @@ private predicate isInvalidFunction(Function func) {
     expr.getEnclosingFunction() = func and
     not exists(expr.getType())
   )
+  or
+  exists(Location l1, Location l2 |
+    l1 = func.getEntryPoint().getLocation() and l2 = func.getEntryPoint().getLocation()
+  |
+    l1 != l2
```

Comment:
Would it be nicer to write

```ql
func.getEntryPoint().getLocation() != func.getEntryPoint().getLocation()
```

or

```ql
strictcount(func.getEntryPoint().getLocation()) > 1
```

or

```ql
count(func.getEntryPoint().getLocation()) != 1
```

(the latter assumes that 0 locations is also fine but I don't know if this ever comes up).

Reply:
Fixed.